run.sh

```diff
@@ -15,17 +15,14 @@
 # limitations under the License.
 #
 
-set -e
+set -ex
```

Comment:
I'd prefer not to have `-x` on by default.

Reply:
Done! (Via phone :O)
I love it for debugging but it is pretty noisey